WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -418,6 +418,39 @@ GET fees
 
 http://www.wasabiwallet.io/
 
+# 9. Setup Cloudflare Anti-DDoS for website
```

Comment:
Can you move it to the end of the document under additional settings? I would add the word "ClearNet" website.